release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2828,3 +2828,26 @@ link:https://access.redhat.com/solutions/6957131[{product-title} 4.9.32 containe
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-9-33"]
+=== RHBA-2022:2206 - {product-title} 4.9.33 bug fix and security update
+
+Issued: 2022-05-18
+
+{product-title} release 4.9.33, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:2206[RHBA-2022:2206] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:2205[RHSA-2022:2205] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6958609[{product-title} 4.9.33 container image list]
+
+[id="ocp-4-9-33-bug-fixes"]
+==== Bug fixes
+
+* Previously, an Application Platform Interface (API) for customizing the console and OAuth platform routes restricted users from specifying custom host names in the cluster ingress configuration and names with top-level domains that included decimal digitals. With this update, users can now use the top-level cluster domains containing decimal digits and customize the console and OAuth routes using any host names that are valid for routes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2075551[*BZ#2075551*])
```

Comment:
```suggestion
* Previously, an Application Platform Interface (API) for customizing the console and OAuth platform routes restricted users from specifying:
** Custom host names in the cluster ingress configuration.
** Names with top-level domains that included decimal digitals.
+
With this update, users can now use the top-level cluster domains containing decimal digits and customize the console and OAuth routes using any host names that are valid for routes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2075551[*BZ#2075551*])
```

Reply:
Purely a formatting suggestion. As written, I stumbled over the sentence a few times.